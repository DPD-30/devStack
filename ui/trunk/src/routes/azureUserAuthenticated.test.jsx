import React from 'react'
import { BrowserRouter as Router } from 'react-router-dom'
import { renderToString } from 'react-dom/server'


import RoutingAzureUserAuthenticated from './azureUserAuthenticated.jsx'

jest.unmock('./azureUserAuthenticated.jsx')


test('RoutingAzureUserAuthenticated matches snapshot', () => {
	const comp = renderToString(<Router><RoutingAzureUserAuthenticated /></Router>)

	expect(comp).toMatchSnapshot()
})
