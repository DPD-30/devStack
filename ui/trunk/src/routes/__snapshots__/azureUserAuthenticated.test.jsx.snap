// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RoutingAzureUserAuthenticated matches snapshot 1`] = `""`;
